README.md

```diff
@@ -5,6 +5,7 @@ a text parser that will attempt to export a text encoded composer symphony, to w
 REQUIREMENTS
 needs python3
 pip3 install edn_format
+pip3 install requests
```

Comment:
[note to self] replace this with `pip3 install -r requirements.txt` and add `requests`